GitTfs/Core/TfsInterop/TfsPlugin.cs

```diff
@@ -20,10 +20,10 @@ public static TfsPlugin Find()
                 return x.Try("GitTfs.Vs" + explicitVersion, "Sep.Git.Tfs.TfsPlugin") ??
                        x.Fail("Unable to load TFS version specified in GIT_TFS_CLIENT (" + explicitVersion + ")!");
             }
-            return x.Try("GitTfs.Vs2015", "Sep.Git.Tfs.TfsPlugin") ?? 
-                   x.Try("GitTfs.Vs2013", "Sep.Git.Tfs.TfsPlugin") ??
+            return x.Try("GitTfs.Vs2013", "Sep.Git.Tfs.TfsPlugin") ??
                    x.Try("GitTfs.Vs2012", "Sep.Git.Tfs.TfsPlugin") ??
                    x.Try("GitTfs.Vs2010", "Sep.Git.Tfs.TfsPlugin") ??
+                   x.Try("GitTfs.Vs2015", "Sep.Git.Tfs.TfsPlugin") ??
```

Comment:
What do you think about leaving Vs2015 at the top of this list if it looks like VS2015 is installed on this computer? (This is a duplicate of the suggestion that I included in https://github.com/git-tfs/git-tfs/issues/893#issuecomment-166050139).